### ISSUE TITLE
Invalid Selector: TypeError: Class constructor ObjectId cannot be invoked without 'new'

### ISSUE THREAD

**Issue by ghengeveld** [mingo-app/mingo#612]
Mingo version: 1.13.4

I get this when running a query like:

```
{ _id: ObjectId("6528fde36417a3572b58e07f") }
```

This can be resolved by adding `new`:

```
{ _id: new ObjectId("6528fde36417a3572b58e07f") }
```

However, this breaks when following a relation through this link:

<img width="250" alt="Screenshot 2023-10-13 at 10 46 25" src="https://github.com/mingo-app/mingo/assets/321738/e08e6436-6f23-4c37-a2e0-555f2645d52c">

This yields an empty screen and the error `Invalid Selector: TypeError: Class constructor ObjectId cannot be invoked without 'new'`.

**Comment by tothradoslav** [comment]
Thank you for letting us know, we corrected it. Will be in new release.

**Comment by pepkin88** [comment]
I'd like to request a clarification.
I hope that the correction involves not requiring the `new` keyword, not just fixing broken relations. Other tools often export lists of ObjectIds without `new`, it would be a shame to be required to specially format them every time, since it worked just fine a version ago.

**Comment by tothradoslav** [comment]
Sure, in queries, you won't have to enter "new". I'll make sure you don't have to enter it in other situations, either.

**Comment by tothradoslav** [comment]
Hi, this has been released in the latest version. Please check it out and let us know if there's still an issue. Thank you!

**Comment by sagrawal31** [comment]
All of a sudden, opening a new collection tab resulting in the same error
![image](https://github.com/mingo-app/mingo/assets/1804514/b27a80d6-2db6-42fa-a060-2af90479670c)

And I can't see anything on `Documents` tab.

I'm on version `1.13.5`. Tried clearing all history as well.

**Comment by tothradoslav** [comment]
Can you please tell me how exactly did you open the tab? From the sidebar?

**Comment by sagrawal31** [comment]
I tried to open from both ways- 1) Via sidebar and 2) Via `CMD + T`

**Comment by sagrawal31** [comment]
Any update here, please? I can't delete the app completely to attempt a fix.

**Comment by pepkin88** [comment]
> Sure, in queries, you won't have to enter "new". I'll make sure you don't have to enter it in other situations, either.

Unfortunately, it seems that `new` is still required. After entering such query:
```
{user: ObjectId("615ad6d95c92ea37d66936a2")}
```
I get this on version 1.13.5:

<img width="594" alt="image" src="https://github.com/mingo-app/mingo/assets/1028647/0a406846-395e-423e-a522-d44c704ae284">

**Comment by sagrawal31** [comment]
Can we open this issue at least?

**Comment by fers4t** [comment]
Still, we have this error.

**Comment by tothradoslav** [comment]
This issue is fixed / available in the new release v1.14.1

https://github.com/mingo-app/mingo/releases/tag/v1.14.1

Please reopen this issue if necessary.
